scripts/byron-to-alonzo/mkfiles.sh

```diff
@@ -319,12 +320,41 @@ echo
 ls -1 shelley/*
 echo "====================================================================="
 
-echo "Generated genesis.json:"
+echo "Generated shelley/genesis.json:"
 echo
 cat shelley/genesis.json
 echo
 echo "====================================================================="
 
+# TODO alonzo: Should this be generated by cardano-cli genesis create?
+
+mkdir -p alonzo
+cat > alonzo/genesis.json <<EOF
+{
+  "adaPerUTxOWord": 0,
+  "executionPrices": {
+    "prMem": 1,
+    "prSteps": 1
+  },
+  "maxTxExUnits": {
+    "exUnitsMem": 1,
+    "exUnitsSteps": 1
+  },
```

Comment:
Once `cardano-cli genesis create` is modified to create an alonzo genesis file, we can remove this.

Reply:
Removed!